### PR TITLE
PCHR-2712: Update option group mocks

### PR DESCRIPTION
**Pull request by igorpavlov**
## Overview

This PR is technical and simply adds Option Group mocks (needed for testing purposes) as per https://github.com/civicrm/civihr/pull/2187.

## Technical Details

Added mocks:

```js
hrleaveandabsences_absence_type_calculation_unit: [
  {
    id: '1169',
    option_group_id: '149',
    name: 'days',
    label: 'Days',
    value: '1',
    weight: '1',
    is_active: '1',
    is_reserved: '1',
    option_group_name: 'hrleaveandabsences_absence_type_calculation_unit'
  },
  {
    id: '1170',
    option_group_id: '149',
    name: 'hours',
    label: 'Hours',
    value: '2',
    weight: '2',
    is_active: '1',
    is_reserved: '1',
    option_group_name: 'hrleaveandabsences_absence_type_calculation_unit'
  }
]
```

---

- ✅ JS distributive files - included
- ⏹ CSS distributive files - not needed
- ✅ Jasmine Tests - passed
- ⏹ Backstop Tests - not needed
- ⏹ PHP Unit Tests - not needed

### CODE REVIEW

File: org.civicrm.reqangular/test/mocks/services/api/option-group-mock.js [hunk 1]
```diff
@@ -129,20 +129,44 @@ define([
               id: '2',
               option_group_id: '22',
               label: 'Region #2',
-              value: 'Region #2',
+              value: '1',
```
Comment (reneolivo) [inline org.civicrm.reqangular/test/mocks/services/api/option-group-mock.js]: why did you change this value and not the others? should this have been changed at all?

Comment (igorpavlov) [reply org.civicrm.reqangular/test/mocks/services/api/option-group-mock.js]: > should this have been changed at all?

This value should be consistent, numeric string.

image

Updated mocks.

**Comment by igorpavlov** [comment]
Updated mocks so they match real responses.
